Core/Object Arts/Dolphin/MVP/Views/Common Controls/TreeView.cls

```diff
@@ -496,6 +496,16 @@ isExpandAllEnabled: aBoolean
 
 	tvFlags := tvFlags mask: DisableExpandAllMask set: aBoolean not!
 
+isExpanded: anObject
+	"Answers if the node associated with anObject is expanded."
+
+	| hItem tvItem dwState |
+	hItem := self handleFromObject: anObject ifAbsent: [^false].
+	tvItem := self getItemState:  hItem.
+	dwState := tvItem dwState.
+	^(dwState bitAnd: TVIS_EXPANDED) == TVIS_EXPANDED
```

Comment:
Thanks. I thought there would be a better way but didn't look very hard. I'll make the change so that the base code has good style.